eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -98,21 +99,38 @@ jobs:
           /p:ServiceDirectory=${{ parameters.ServiceDirectory }}
           /p:IncludeSrc=false /p:IncludeSamples=false
           /p:BuildInParallel=${{ parameters.BuildInParallel }}
+          /p:AutoUpdateSessionRecords=true
           $(ConvertToProjectReferenceOption)
 
         displayName: "Build & Test (all tests for $(TestTargetFramework))"
         env:
+          ${{if eq(variables['Record'], 'true')}}:
+            AZURE_TEST_MODE: "Record"
```

Comment:
Tip: quotes aren't required in YAML unless they are for invalid decimals (e.g. version) or the string starts with a character like `*` or a number.